### ISSUE TITLE
Support for PDF diffs

### ISSUE THREAD

**Issue by Sparticuz** [tomoki1207/vscode-pdfviewer#70]
Looking at a diff of a PDF gives me this:
![image](https://user-images.githubusercontent.com/161913/88336372-e0996780-cd02-11ea-83a6-1525f7f10d54.png)

**Comment by bersbersbers** [comment]
Interesting, I see something different: When I open a PDF file from the source control tab, I see the new pdf twice (while I had expected to see the old PDF file next to the new PDF file).

**Comment by bersbersbers** [comment]
> Interesting, I see something different: When I open a PDF file from the source control tab, I see the new pdf twice (while I had expected to see the old PDF file next to the new PDF file).

This is true only with Remote SSH. In a local directory, I am seeing something similar to @Sparticuz on the left (old) side:

![image](https://user-images.githubusercontent.com/12128514/106583647-9ca2dd00-6545-11eb-9de0-f0da5f50301e.png)

**Comment by bersbersbers** [comment]
I filed https://github.com/microsoft/vscode/issues/115614
